source/module_elecstate/elecstate.cpp

```diff
@@ -406,6 +408,9 @@ void ElecState::init_ks(Charge* chg_in, // pointer for class Charge
     // initialize ekb and wg
     this->ekb.create(nk_in, GlobalV::NBANDS);
     this->wg.create(nk_in, GlobalV::NBANDS);
+
+    this->nbz = nbz_in;
```

Comment:
Should nbz and bz become members of elecstate? They are members of bigpw and I remember comments say that bigpw will be removed in due course. @dyzheng

Reply:
Another choice is to add bigpw to the member of Elecstate directly.

Reply:
I have added bigpw as the member of Elecstate.